community/tutorials/gdnative/gdnative-c-example.rst

```diff
@@ -0,0 +1,184 @@
+.. _doc_gdnative_c_example:
+
+GDNative C example
+==================
+
+Introduction
+------------
+This tutorial will introduce you to the bare minimum required to create GDNative modules. This should be your starting point into the world of GDNative, understanding the contents of this tutorial will help you in understanding all that is to come after this.
```

Comment:
Maybe it would help to explain the GDNative/nativescript/PluginScript … distinctions first? Otherwise, I guess, it may be confusing. E.g. I found some explanation in the `PluginScript` commit/PR:
 “It abstracts away some of the inconviences of GDNative. If a language is exposed as a PluginScript it will benefit from the same editor integration as GDScript/C#. So you can download the plugin from the assetlib and are able to choose that language directly in the “Add script” dialog, have the buildsystem set up, use the godot script editor, able to use templates, build automatically when running the game, etc.. all the things you'd have to do manually when using GDNative bindings directly.”